lib/link.js

```diff
@@ -41,6 +42,10 @@ export default class Link extends Component {
   }
 
   render () {
+    if (Children.count(this.props.children) > 1) {
+      warn(`Warning: 'Link' accepts only one child element.`)
+    }
```

Comment:
Tried both, using `React.Children.only` will kill the `<Link href="/">test</Link>` use case, since it forces the children to be a React element, which a string is not. Also tried adding 
```
  static propTypes () {
    return {
      children: PropTypes.oneOfType([
        PropTypes.string,
        PropTypes.element
      ]).isRequired
    }
  }
```

but for some reason it still passes
```
<Link href="/">
  <h1>test</h1>
  <h2>test2</h2>
</Link>
```

In the end the solution I PR'ed seems to be one of the few solutions. We could change it to a custom PropType validation function though.